site.yml

```diff
@@ -0,0 +1,15 @@
+runtime:
+  cache_dir: ./.cache/antora
+site:
+  title: InfoQ Mini Book
+content:
+  sources:
+    - url: '.'
+      branches: HEAD
+      start_path: src/docs/asciidoc
+ui:
+  bundle:
+    url: https://github.com/Mogztter/mini-book-default-ui/releases/download/v0.2.1/ui-bundle.zip
+    snapshot: true
```

Comment:
I'm guessing this will eventually change to a released version?

Reply:
This is a released version, the artifact (zip) is built from a tag/release using GitHub Actions: https://github.com/Mogztter/mini-book-default-ui/blob/master/.github/workflows/release.yml